### PR TITLE
Update thor-upstream

### PR DESCRIPTION
**Pull request by otherview**
# Description

Updates thor-upstream .

Fixes # [(issue)](https://github.com/vechain/protocol-board-repo/issues/164)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] New and existing E2E tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have not added any vulnerable dependencies to my code

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -22,6 +22,7 @@ require (
 	github.com/pkg/errors v0.8.0
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/prometheus/client_golang v1.18.0
+	github.com/prometheus/client_model v0.5.0
```
Comment (otherview) [inline go.mod:25]: I guess a `go mod tidy` was missed in one of the latest metrics PR ?

Comment (libotony) [reply go.mod:25]: Here https://github.com/vechain/thor/pull/758/files

**Comment by libotony** [comment]
I would like to postpone the update of this dependency, there will be updates for it as a target for the next release.

**Comment by otherview** [comment]
Putting this under a pin - if there are other PR's wrt go-ethereum then use those instead of this one for the release.

**Review by libotony — APPROVED** [review]

**Pull request merged**